api/redhatopenshift/HcpCluster/hcpCluster-models.tsp

```diff
@@ -218,6 +214,21 @@ model ApiProfile {
   visibility: Visibility;
 }
 
+/** Information about how Etcd is Encrypted. */
+model EtcdEncryptionProfile {
```

Comment:
I understand this encryption is intended to be etcd data-level encryption https://docs.openshift.com/container-platform/4.15/security/encrypting-etcd.html and not disk-level encryption, right?